src/main/java/io/github/moulberry/notenoughupdates/overlays/MiningOverlay.java

```diff
@@ -440,11 +441,13 @@ public void update() {
 			}
 			if (forgeDisplay) {
 				overlayStrings = new ArrayList<>();
-				if (NotEnoughUpdates.INSTANCE.config.mining.forgeDisplayEnabledLocations == 1 &&
-					!SBInfo.getInstance().isInDungeon) {
-					overlayStrings.addAll(getForgeStrings(profileConfig.forgeItems));
-				} else if (NotEnoughUpdates.INSTANCE.config.mining.forgeDisplayEnabledLocations == 2) {
-					overlayStrings.addAll(getForgeStrings(profileConfig.forgeItems));
+				if (!NotEnoughUpdates.INSTANCE.config.mining.forgeDisplayOnlyShowTab || Keyboard.isKeyDown(Keyboard.KEY_TAB)) {
```

Comment:
what if someone has changed the keybind

Reply:
I wrote "tab key" not "opening the tab list" so I dodged a bullet there. Do you know how to access the vanilla minecraft keybild?

Reply:
GameSetting.keyBindPlayerList

Reply:
This does not react instantly
`Minecraft.getMinecraft().gameSettings.keyBindPlayerList.isKeyDown()`

but this does.
`Keyboard.isKeyDown(Minecraft.getMinecraft().gameSettings.keyBindPlayerList.getKeyCode())`

strange world i guess?

Reply:
KeyBind#isKeyDown vs Keyboard#isKeyDown is also checking for collisions i think